cmd/ci-operator/main.go

```diff
@@ -943,6 +892,57 @@ func (o *options) initializeNamespace() error {
 		}
 	}
 
+	ssarStart := time.Now()
```

Comment:
Can we not simply set the annotation on creation or does the projects api not allow that?

Reply:
Possibly, haven't tried. It's not entirely braindead because the annotations are technically optional and you may still need to update afterwards because you are reusing an existing project but it would be a safest thing to do.

I'm not too keen on doing it myself though - reordering stuff seemed easy enough to just do it, but I'd really like to avoid another rabbit hole :)

Reply:
I guess the point is, this might just fail because the SSAR succeeded and is generally still prone to racing

Reply:
The ProjectRequest takes an ObjectMeta, lets just add the annotation there and then do the update after SSAR. We still have to do the update in case we didn't create the project

Reply:
OK, will fixup the PR, stay tuned.

Reply:
You can't set annotations on project (historical limitation).  So if you send them they'll be ignored I think (but if you see in your test they don't let me know)

Reply:
Let's add a comment, when I wrote this it was common knowledge